src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -279,14 +275,18 @@ private void setupKeyboardShortcuts() {
                 }
             }
             if (MANAGE_ASSIGNEES.match(event) && ui.getBrowserComponent().isCurrentUrlIssue()) {
-                ui.getBrowserComponent().switchToTab(DISCUSSION_TAB);
+                if (ui.getBrowserComponent().isCurrentUrlPR()) {
+                    ui.getBrowserComponent().openPrTab(getCurrentlySelectedIssue(), DISCUSSION_TAB);
+                }
                 ui.getBrowserComponent().manageAssignees(event.getCode().toString());
             }
             if (SHOW_MILESTONES.match(event)) {
                 if (KeyPress.isValidKeyCombination(GOTO_MODIFIER.getCode(), event.getCode())) {
                     ui.getBrowserComponent().showMilestones();
                 } else if (ui.getBrowserComponent().isCurrentUrlIssue()) {
-                    ui.getBrowserComponent().switchToTab(DISCUSSION_TAB);
+                    if (ui.getBrowserComponent().isCurrentUrlPR()) {
+                        ui.getBrowserComponent().openPrTab(getCurrentlySelectedIssue(), DISCUSSION_TAB);
```

Comment:
More importantly, I think there might be a bug here in the original implementation (you should try it out first, see if my hypothesis is correct) where we might jump to `manageMilestones` before the PR discussion tab is opened. If that is the case, you need to use `waitUntilDiscussionPageLoaded`.